CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## v8.0.3 - 2025-08-12
+
+This is only relevant for Kotlin.
+Fixes page size alignment for all supported linkers.
+Adds `ClientId.copyBytes()` for Kotlin.
```

Comment:
For v8.0.2 we said "only relevant to Android", here we're mentiong "for Kotlin" -- did the change in v8.0.2 also impact non-Android Kotlin users (JVM)?

Reply:
It also affects JVM since the wrapper is shared between Android and JVM so this is more correct.